### ISSUE TITLE
Login and Register component selectors aren't recognized

### ISSUE THREAD

**Issue by andrew-rice-p33** [AnthonyNahas/ngx-auth-firebaseui#415]
### Bug Report or Feature Request (mark with an `x`)
```
- [x] bug report -> please search issues before submitting
- [ ] feature request
```
### OS and Version?
> macOS Sierra

### Versions
Angular CLI: 8.3.21
Node: 12.4.0
OS: darwin x64
Angular: 8.2.14
... animations, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.803.21
@angular-devkit/build-angular     0.803.21
@angular-devkit/build-optimizer   0.803.21
@angular-devkit/build-webpack     0.803.21
@angular-devkit/core              8.3.21
@angular-devkit/schematics        8.3.21
@angular/cdk                      8.2.3
@angular/cli                      8.3.21
@angular/fire                     5.2.3
@angular/flex-layout              8.0.0-beta.27
@angular/material                 8.2.3
@ngtools/webpack                  8.3.21
@schematics/angular               8.3.21
@schematics/update                0.803.21
rxjs                              6.4.0
typescript                        3.5.3
webpack                           4.39.2

### Repro steps

After following all the steps for installation from this repository [(found here)](https://ngx-auth-firebaseui.firebaseapp.com/getting-started) I am able to use all of the components except for ngx-auth-firebaseui-login and ngx-auth-firebaseui-register. The only notable difference is I used the short install from the repository's readme and ran ng add ngx-auth-firebaseui. The log is posted below.

### The log given by the failure
compiler.js:2175 Uncaught Error: Template parse errors:
'ngx-auth-firebaseui-login' is not a known element:
1. If 'ngx-auth-firebaseui-login' is an Angular component, then verify that it is part of this module.
2. If 'ngx-auth-firebaseui-login' is a Web Component then add 'CUSTOM_ELEMENTS_SCHEMA' to the '@NgModule.schemas' of this component to suppress this message. ("[ERROR ->]<ngx-auth-firebaseui-login></ngx-auth-firebaseui-login>

Was this functionality removed? Looking at the package in node_modules it doesn't have the login and register components.

**Comment by andrew-rice-p33** [comment]
I have it working now, but it is still acting odd. 

I started getting the following error in my console running ng serve:
`ERROR in node_modules/ngx-auth-firebaseui/ngx-auth-firebaseui.d.ts(7,45): error TS2305: Module '"./module/interfaces/config.interface"' has no exported member 'ngxAuthFirebaseUIConfigFactory'.`

I've seen a TS2305 before, so I did what I did then and deleted node_modules and reinstalled with npm i --save. It still didn't work. So then I decided to run ng add ngx-auth-firebaseui again. After cleaning up the duplicate imports in my app.module.ts, VSCode was still telling me ngx-auth-furebaseui-login didn't exist and, even more worrying was the selectors that worked before weren't recognized. I tried it anyway and, strangely, it is working. I'm not sure exactly what's going on, but it may still be worth looking into. I'm happy to try other things to diagnose what happend.
